MAUI/Autocomplete/No-Results-Found.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: No Result Found in in Autocomplete control | Syncfusion
+description: Learn about No Result Found support in .NET MAUI Autocomplete (SfAutocomplete) control and more details.
+platform: maui
+control: SfAutoComplete
+documentation: ug
+---
+# No Result Found in .NET MAUI Autocomplete  (SfAutoComplete)
```

Comment:
use -> No Results Found in .NET MAUI Autocomplete (SfAutocomplete)

Reply:
updated.